src/modelbench/benchmark_runner.py

```diff
@@ -189,9 +192,23 @@ def __init__(self, runner: "TestRunnerBase"):
         self.test_records = defaultdict(dict)
 
     def add_test(self, test: PromptResponseTest):
+        assert self._test_lookup.get(test, None) is None, f"Test {test.uid} already added"
```

Comment:
Is it a bad thing if there are duplicates in tests? If so, that should be solved at the TestRunner or TestRunnerBase level.

Reply:
I think we should catch if duplicate tests are added, I don't see any scenarios where that would be intentional.
I did want to put the check in the runner originally, but the BenchmarkRunner and TestRunner don't add tests the same way so there would be duplicate code.

Reply:
I'm still not seeing the bad thing. If somebody accidentally adds it twice, can't we just be idempotent and ignore the second one? Generally the closer we get to a human user, the more I think code should be forgiving and honor DWIM.

If we do have to throw an error because what they're asking for doesn't make sense, since somebody using the BenchmarkRunner won't be adding tests at all, I think we could just put a human-focused error message in the TestRunner, so that they are alerted to the problem immediately. If there's a problem, that is.

Reply:
I see your point, I'll remove this assertion!